### PR TITLE
Updated Ubuntu source creation commands

### PR DESCRIPTION
**Pull request by M-D-M**
Updated the Ubuntu source creation commands to remove need for piping

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@M-D-M Thanks for your contribution! There are a few more instances of `curl ... | sudo tee ...` remaining in the same file, do you mind update them all?
It would be great if you update all instances, but this is not a blocking comment.

**Comment by M-D-M** [comment]
Sure, let me take a look -- I made sure only to update commands for Ubuntu so far, since it's what I use mostly.

**Comment by M-D-M** [comment]
@daxian-dbw Daxian, I took a look at the other commands for the other distros.  Debian uses a slightly different command so I left it alone.  I don't have access to the other distros (CentOS, RHEL) to check -- the version of curl that ships with those might be slightly different, so I left them alone as well.

**Comment by daxian-dbw** [comment]
@M-D-M Thanks for looking into it! I appreciate your effort.

File: docs/installation/linux.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This is the preferred method.
 curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
 
 # Register the Microsoft Ubuntu repository
-curl https://packages.microsoft.com/config/ubuntu/14.04/prod.list | sudo tee /etc/apt/sources.list.d/microsoft.list
+sudo curl -o /etc/apt/sources.list.d/microsoft.list https://packages.microsoft.com/config/ubuntu/14.04/prod.list
```
Comment (iSazonov) [inline docs/installation/linux.md]: Should we remove 14.04 - the version is out of support?

Comment (M-D-M) [reply docs/installation/linux.md]: I had a hosting provider that will go unnamed that _just_ upgraded a VPS from 14.04.  It's EOL from Canonical, but there's definitely some users out there still on it. :p

**Pull request merged**